Cargo.toml

```diff
@@ -15,9 +15,13 @@ publish = false
 targets = ["x86_64-pc-windows-gnu", "x86_64-pc-windows-msvc"]
 
 [dependencies]
-rustls = { git = "https://github.com/rustls/rustls.git", default-features = false, features = [
-    "std",
-] }
+rustls = { version = "0.23.0", features = ["ring", "tls12", "std"], default-features = false }
```

Comment:
Remove the ring and tls12 features, they are controlled by the matching feature flags in rustls-cng.

Reply:
Thanks for the comments! appreciate it! We just sent email to you, hope you get it! Sorry we didn't contact you earlier for the thoughts of using this craft. This is just a preliminary draft. I will address your comments in my code. Thanks again!

Reply:
your new commit already has it. so I won't touch cargo.toml.